web/src/components/block-content/iframe.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { jsx, Text } from "theme-ui";
+
+function IFrame(props) {
+  return (
+    <div style={{ alignItems: "center" }}>
+      <br />
+      {/* 56.25% is for 16:9 aspect ratio */}
+      <div style={{ paddingBottom: "56.25%", position: "relative" }}>
```

Comment:
Can you explain how whether the content below the iframe is rendered? (i.e. does it show up on top of the bottom padding?)

Also, I would be inclined to add "aspectRatio" as a field to Sanity and default to 16/9 instead of fully hardcoding here.

Reply:
Nope, does not show up on top of bottom padding as a result of the absolutely positioned child. Also, yeah I can add the aspect ratio as a field in Sanity!